resources/schema.json

```
@@ -307,15 +307,33 @@
                     {
                       "required": [
                         "variables"
-                      ]
+                      ],
+                      "not":
+                      {
```

```suggestion
                      "not": {
```

Same for all other occurences below

---

thansk, fixed